.pfnci/config.pbtxt

```diff
@@ -167,3 +167,25 @@ configs {
     command: "bash .pfnci/script.sh py37.amd"
   }
 }
+
+# Use CUB
+configs {
+  key: "cupy.py37.cub"
+  value {
+    requirement {
+      cpu: 8
+      memory: 24
+      disk: 10
+      gpu: 2
+    }
+    time_limit {
+      seconds: 1800
+    }
+    checkout_strategy {
+      include_dot_git: true
+    }
+    environment_variables { key: "GPU" value: "2",
+                            key: "CUB_DISABLED" value:"0"}
```

Comment:
You probably already knew: `CUB_DISABLED` will have no effect once #2584 is merged, so this key can be removed afterwards. (Keeping it here has no harm though.)

Reply:
I actually think is better to keep it around, because it will simplify CI.